boranga/components/species_and_communities/api.py

```diff
@@ -139,6 +145,10 @@ def species_internal(self, request, *args, **kwargs):
 class CommunitiesFilterBackend(DatatablesFilterBackend):
     def filter_queryset(self, request, queryset, view):
         total_count = queryset.count()
+        # filter_group_type
+        # filter_group_type = request.GET.get('filter_group_type')
+        # if filter_group_type:
+        #     queryset = queryset.filter(group_type__name=filter_group_type)
         # filter_community_id
```

Comment:
Are you able to delete this commented code?

Reply:
This is required but, I wasn't having the updated models to apply this, so I have commnmeted . As we have now added grouptype_id in Community model.